generators/component/templates/component.ts

```diff
@@ -46,6 +88,11 @@ export class <%= component.class %> extends<% if (component.isExtendingMes) { %>
 
     //#endregion
 
+    /**
+     * Constructor
+     * 
+     * @param viewContainerRef the reference to the component view container
+     */
     constructor(viewContainerRef: ng.ViewContainerRef) { 
```

Comment:
@jpsfs also the elementRef is stores in CoreComponent.framework.elementRef, to be used by customization if needed. This should be changed to elementRef anyway.